### PR TITLE
Directory data format

### PR DESCRIPTION
**Pull request by rtilder**

### CODE REVIEW

**Comment by rtilder** [comment]
@fmarier r?

File: shavar/tests/test_sources.py [hunk 1]
```diff
@@ -48,19 +60,44 @@ def test_list_chunks(self):
...
+
+    def test_no_data(self):
+        d = DirectorySource("dir://tarantula", 1)
+        self.assertRaises(NoDataError, d.load)
+
+
 class TestS3FileSource(ShavarTestCase):
 
     # I have no idea where I came up with this name or what it might mean but
     # it's a keeper.
```
Comment (fmarier) [inline shavar/tests/test_sources.py]: +1 :)

**Comment by fmarier** [comment]
At a high-level, that looks good to me. Maybe @groovecoder wants to take a look too since he's going to become intimate with this code soon.

**Comment by rtilder** [comment]
Going to merge now so I can tag for QA.  This code won't actually go into use any time very soon so I'm not terribly worried about it.

**Pull request merged**